desc/external/paraview.py

```diff
@@ -0,0 +1,260 @@
+"""Utility functions to export DESC objects to Paraview."""
+
+import warnings
+
+import numpy as np
+
+try:
+    import pyvista as pv
+    from pyvista import CellType
+except ImportError:
+    warnings.warn(
+        "DESC objects are exported to Paraview using `pyvista`"
+        "package which is an optional dependency. Please pip install "
+        "`pyvista` to your environment."
+    )
+
+from desc.coils import CoilSet, _Coil
+from desc.equilibrium import Equilibrium
+from desc.geometry import FourierRZToroidalSurface
+from desc.grid import LinearGrid
+
+
+def export_surface_to_paraview(
+    obj, res=(100, 100), keys=[], rho=1.0, filename="surface"
+):
+    """Export a constant rho surface as VTU file.
+
+    Parameters
+    ----------
+    obj : Equilibrium or FourierRZToroidalSurface
+        Object to be exported. Different flux surfaces of the Equilibrium
+        can be exported by supplying `rho`.
+    res : tuple, list
+        The resolution used to create the mesh. Must be in the form
+        (N_toroidal, N_poloidal). Default is (100, 100).
```

Comment:
I'd use `M_poloidal` here, and maybe switch the order to make it match other places in the code